### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#46]
```
Translating Keepnote in other languages (french)


```

Original issue reported on code.google.com by `comze...@gmail.com` on 27 Apr 2009 at 6:29

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Hold on it’s coming (after version 0.5.3)

BTW, beware that translation disclaimer
It seems to guarantee that the FSF (and everyone else) will always be allowed to
freely distribute these translations
http://translationproject.org/html/whydisclaim.html
```

Original comment by `thibaut....@gmail.com` on 19 Jun 2009 at 12:51

**Comment by GoogleCodeExporter** [comment]
```
French is now in KeepNote 0.5.3.  If you are interested in contributing other
language translations please contact me.
```

Original comment by `matt.rasmus@gmail.com` on 8 Jul 2009 at 12:42
- Changed state: **Done**

**Comment by GoogleCodeExporter** [comment]
```
Hi,

I installed version 0.5.3, but I don't see any change. How can I get French?

Thanks.
```

Original comment by `cont...@jpfleury.net` on 19 Jul 2009 at 4:37

**Comment by GoogleCodeExporter** [comment]
```
I didn't properly package KeepNote 0.5.3.  So unfortunately French translations 
are
not showing up completely.  I will do more testing and make sure they are 
included in
the next release.
```

Original comment by `matt.rasmus@gmail.com` on 20 Jul 2009 at 5:18
- Changed state: **Started**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
French and Turkish will be available in KeepNote 0.6
```

Original comment by `matt.rasmus@gmail.com` on 3 Sep 2009 at 12:34
- Changed state: **Fixed**
